numpyro/primitives.py

```diff
@@ -137,8 +137,6 @@ def param(name, init_value=None, **kwargs):
         'args': (init_value,),
         'kwargs': kwargs,
         'value': None,
-        'mask': None,
-        'scale': None,
```

Comment:
We are not yet using `MaskedDistribution` in MaskMessenger in Pyro. Are there any issues that this might cause when integrating with funsor? See - https://github.com/pyro-ppl/pyro/pull/2284. cc @fritzo.

Reply:
I am not sure. In [enum PR](https://github.com/pyro-ppl/numpyro/pull/572/files#diff-fee397191e0cc75a2e33eea8f7ecb6d5R186), `mask` is used to mask out the log_prob term, as in the usual `log_density` implementation. I am not sure about the following too: "automatic broadcasting to the mask shape when the mask is enumerated". AFAIK we just use `mask` for log_prob computation and that job is handled in MaskedDistribution.log_prob method.